Makefile

```diff
@@ -262,7 +262,7 @@ protobuf: check-protoc
 		echo ; \
 		yarn --silent pbjs \
 			../proto/streamlit/proto/*.proto \
-			--path=proto -t static-module --wrap es6 \
+			--path=../proto -t static-module --wrap es6 \
```

Comment:
it's been a while since I made this commit on this branch -- and I didn't exactly leave myself the most clear commit message. But IIRC this was required with the upgrade to ES6. I'll do some testing and if it still works the old way I'll roll it back.

Reply:
I checked with the team and we don't see any thing blocking the upgrade. So please feel free to publish a separate PR! We can get that merged prior to this one.